tests/utils.go

```diff
@@ -805,8 +805,14 @@ func AdjustKubeVirtResource() {
 	PanicOnError(err)
 	KubeVirtDefaultConfig = adjustedKV.Spec.Configuration
 	CDIInsecureRegistryConfig, err = virtClient.CoreV1().ConfigMaps(flags.ContainerizedDataImporterNamespace).Get(insecureRegistryConfigName, metav1.GetOptions{})
```

Comment:
@oshoval what I meant, this call here

```
	CDIInsecureRegistryConfig, err = virtClient.CoreV1().ConfigMaps(flags.ContainerizedDataImporterNamespace).Get(insecureRegistryConfigName, metav1.GetOptions{})
```

throwing an error or not, will always return a ConfigMap (that is due to how the function is written). Therefore it can never be nil. Spmething like this would work:


```
	if err != nil {
		if errors.IsNotFound(err) {
                        // force it to nil, independent of what the client returned
			CDIInsecureRegistryConfig = nil
		} else {
			PanicOnError(err)
		}
```

This change should resolve all issues.

Reply:
Done
one question, dont we need to make sure the configMap didnt gone away after it was existing ?

this makes #4470 not needed i guess
testing locally